hca-api-stub/voa-rest.js

```diff
@@ -74,7 +74,7 @@ function attach(app) {
       connector: require('loopback-connector-soap'),
       remotingEnabled: true,
       wsdl: path.join(__dirname, './voa.wsdl'),
-      url: endpoint.esPreprod,
+      url: 'http://vaausesrapp803.aac.va.gov:7401/voa/voaSvc',
```

Comment:
when we were trying to get this to work @awong-dev told me to replace `url: endpoint.esPreprod` with `url: 'http://vaausesrapp803.aac.va.gov:7401/voa/voaSvc'` so I didn't even look at what was in the block. I can swap this out.